src/Text/Time/Pretty/TimeAgo.hs

```diff
@@ -90,4 +101,4 @@ timeAgoToDiffTime TimeAgo {..} =
          LT -> negate)
         (picoSecondsAgo +
          picoSecondsPerSecond *
-         (secondsAgo + 60 * (minutesAgo + 60 * (hoursAgo + 24 * daysAgo))))
+         (secondsAgo + 60 * (minutesAgo + 60 * (hoursAgo + 24 *( daysAgo + 7 * weeksAgo )))))
```

Comment:
Pleaso don't remove newlines at the end of files.

Reply:
I added the newline.

Reply:
I don't see the new commit?

Reply:
I think the commits should be visible now.